### ISSUE TITLE
Exacte bewoording verwijzen naar een WCAG-SC aanpassen

### ISSUE THREAD

**Issue by rianrietveld** [nl-design-system/documentatie#562]
In plaats van 

> Door het geven van een logische volgorde van informatie voldoe je aan het WCAG-succescriterium [1.3.2 Betekenisvolle volgorde](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) (niveau A).

Zeggen:

> Het geven van een logische volgorde van informatie is nodig om te voldoen aan het WCAG-succescriterium [1.3.2 Betekenisvolle volgorde](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) (niveau A).

Dus:
dus de toevoeging **nodig om te** erbij.

Zie discussie in Slack:
https://codefornl.slack.com/archives/C02CWGL2M18/p1706161264610819

**Comment by rianrietveld** [comment]
Duplicate of #542, deleting this one
